nrf_modem/doc/ug_nrf_modem_porting_os.rst

```diff
@@ -25,37 +24,38 @@ Creating the OS abstraction layer
 *********************************
 
 To create an OS abstraction layer for the Modem library, you must implement the functions in the :file:`nrf_modem_os.h` file.
+An OS abstraction layer implementation for the Zephyr RTOS is also available in the |NCS|.
+See :file:`nrf_modem_os.c` for more details.
+The implementation details for each function are shown in the section following and in the header file:
 
 nrf_modem_os_init()
 ===================
 
 This function is called by the Modem library when the application has issued :c:func:`nrf_modem_init`.
-It is responsible for preparing IRQ for low priority Modem library scheduling and trace scheduling.
+It is responsible for initializing OS-specific functionality related to the Modem library OS abstraction.
 
-.. note::
-   When working with an application based on Zephyr, set the IRQs to a low priority (6 or 7) and enable them before exiting the function.
-
-The function must also initialize the timers and threads (if there is a context that needs a timeout).
-If Nordic Proprietary trace is enabled, the library generates trace data and forwards it to a medium that can be initialized or configured by using the :c:func:`nrf_modem_os_init` function.
-The forwarded trace data is handled in the :c:func:`nrf_modem_os_trace_put` function.
-See :ref:`trace_output_function` for more information.
+If Nordic Proprietary trace is enabled, the library generates trace data that can be retrieved using the :c:func:`nrf_modem_os_trace_get` function.
+See :ref:`trace_handling` for more information.
 
 *Required actions*:
 
 * Initialize timers/threads.
 * Configure low priority Modem library scheduling IRQ (SoftIRQ).
-* Configure low priority trace scheduling IRQ (SoftIRQ).
-* Configure medium for trace (for example, UART or SPI).
+* Initialize heap memory.
+* If Nordic Proprietary trace is enabled, initialize a trace thread and the trace backend(for example, UART or SPI).
+
+nrf_modem_os_busywait()
+=======================
 
+This function is called by the Modem library when a blocking timed context is required.
+
+.. _nrf_modem_os_timedwait:
```

Comment:
There should be a space between lines 52 and 53. This should fix the doc build failure. I am not sure why this was not caught when I ran this in my local build.

Reply:
Thanks